### ISSUE TITLE
Is this project supported anymore?

### ISSUE THREAD

**Issue by Tommigun1980** [molinch/FFImageLoading#1522]
## 🏗 Enhancement Proposal

Hi and thanks for a great plugin that a lot of people rely on!

Unfortunately it doesn't seem that this project is really maintained anymore, which puts us all at risk if this really is the case.
Could a developer please confirm if this project is still alive, and if it is on life support could it be transferred to a new steward?

The reason I am asking is that so many people rely on this and I haven't seen anyone replying to tickets for months. I personally have reported several crash bugs but there is no correspondence, and this worries me.

Looking forward to being informed on the state of FFImageLoading and once again a huge thank you!

**Comment by IainS1986** [comment]
So I guess the answer is maybe, 'No'.

So do people have any other project/repo suggestions for an alternative to FFImageLoading?

**Comment by giuseppenovielli** [comment]
I suggest to moved FFImageLoading into Xamarin Forms Community Toolkit.

I thinks is the best place to contonuos support this project.

**Comment by Tommigun1980** [comment]
Maybe it could be polled what features people use from it and move only
over what people actually use?

(I, for one, only need the caching aspects. Imho Xamarin.Foms's image
should handle that natively out of the box as it's such a common use case.
Iirc it's at least partially accomplishable via image sources, but
FFImageLoading makes it so convenient with its caching properties - just
set up different styles for images that should be cached to disk and for
memory-cached images and off you go. Preloading should also be supported.)

**Comment by giuseppenovielli** [comment]
@daniel-luberda what you thinks?

**Comment by giuseppenovielli** [comment]
@Tommigun1980 i think, i must create a discussion into Xamarin Forms Community Toolkit, and check Microsoft's engineers with community can continue supporting this project, merging into XamarinCommunityToolkit...

I thinks a lot of developers use FFImageLoading with Xamarin.Forms....

**Comment by fdhsdrdark** [comment]
I actually use it but not with Xamarin Forms but in seperate Xamarin Android and Xamarin iOS projects. Xamarin Comunity Toolkit is avaliable only for Xamarin Forms, something not convenient for me and all other users like myself.

**Comment by IainS1986** [comment]
Yes, this shouldn't move to a Forms only repository as it will greatly restrict its use cases. Xamarin Native would need to be supported outside of Toolkit.

However when I posed the question if anyone had alternatives, I was meaning if people had found other solutions or alternatives to drop FFImageLoading. I can't see this getting moved somewhere else and being maintained.

**Comment by fdhsdrdark** [comment]
An alternative would be GlideX but again it's for Forms only. And of course it's not providing everything FFImageLoading provides. 
Additionally there is SquarePicasso, but I had memory issues while testing on old phones, so not very happy with it. 

I believe loading images is something crucial that Xamarin Essentials might be able to provide. 

https://devblogs.microsoft.com/xamarin/glidex-fast-images-android/

https://github.com/xamarin/XamarinComponents/tree/master/Android/SquarePicasso

**Comment by fdhsdrdark** [comment]
I have proposed a feature request for an Image Loading Library in Xamarin Essentials. However, it is not considered the best place for a feature like this. According to jamesmontemagno this "is best to live in Xamarin.Forms and .NET MAUI for UI specifics". Based on his answer, a question: If somethig lives in Xamarin Forms and .NET MAUI will it be available for Xamarin Native also?

https://github.com/xamarin/Essentials/issues/1695

**Comment by giuseppenovielli** [comment]
@nikossaperas Xamarin.Forms is based on Xamarin.Native...

if there is not Xamarin.Native, there is not Xamarin.Forms...

Xamarin.Forms is an architecture to use Xamarin.Native cross-platform....

So your question 'Based on his answer, a question: If something lives in Xamarin Forms and .NET MAUI will it be available for Xamarin Native also?'

No, why Xamarin.Forms has an architecture designed for cross-platform App, but code behind Xamarin.Forms is Xamarin.Native.

**Comment by jslachta** [comment]
@nikossaperas, the question should be put in the XCT (Xamarin Community Toolkit) instead.

**Comment by fdhsdrdark** [comment]
@giuseppenovielli @jslachta 
Ok, thanks very much and sorry about the out of context question.

**Comment by IainS1986** [comment]
> @nikossaperas, the question should be put in the XCT (Xamarin Community Toolkit) instead.

As has already been said. This project supports Xamarin Native. Putting it in XCT reduces the scope of the project significantly and would make it unusable to a lot of developers.

Its should *not* become Xamarin Forms only just so it gets maintained. Especially as internally it would be working natively anyway.

**Comment by rs-mobitech** [comment]
I am reading the suggestions about adding to Xamarin Forms Community Toolkit and wondering.  Is the functionality of this code not already available in ResizetizerNT and would that not be a better source of code?  I thought MAUI was already going to have something to create images in the different projects so is there even any need for a product like this to be added to XFCT?

**Comment by giuseppenovielli** [comment]
@rscholey ResizetizerNT has async image loading?

I'm personally use FFImageLoading for async loading, that is powerful into ListView.
SVG, yes there is ResizetizerNT....

**Comment by ndastur** [comment]
@daniel-luberda I think there are a few people that would be grateful for a reply on this one. We all understand that sometimes projects can't be maintained anymore. Just let us know if you can't support anymore and then also consider handing it on?

**Comment by EmilAlipiev** [comment]
> @rscholey ResizetizerNT has async image loading?
> 
> I'm personally use FFImageLoading for async loading, that is powerful into ListView.
> SVG, yes there is ResizetizerNT....

I thought ResizetizerNT  is not display svg files but generating pngs out of a svg ? did they chance something?

**Comment by giuseppenovielli** [comment]
@EmilAlipiev i never use ResizetizerNT, i don't know

**Comment by EmilAlipiev** [comment]
Has anyone forked and tried to update the Project to the latest versions? I have spent a whole day but project is impossible. I don't get the architecture, all spagetti and project references seems to be cycled. No wonder why there is no single "real" contributor for such large and widely used project. Project is 100% dead and cannot be recovered in my opinion.

**Comment by pictos** [comment]
https://twitter.com/pj_souz/status/1440157986614169611

tl;dr I'm porting this lib to Maui, using the Compatibility package.
I don't have time to take ownership of this lib, since it's pretty huge. My goal here is to provide a version that works with MAUI, that way (I hope) you can port your apps to .NET MAUI.


I would love if someone can test the iOS part

**Comment by molinch** [comment]
Hi all, apologies for getting no news on the status of the lib.
I initially created the lib for Xamarin.Android and Xamarin.iOS, for my usages, then handed over to Daniel, which did all the Xamarin Forms bits and it got successful thanks to him.
I wasn't much involved after that, and don't work anymore with Xamarin/Maui. I also don't think Daniel is doing anything on the project anymore.
If folks like @pictos are happy to to take over please ping me.

**Comment by pictos** [comment]
@molinch sadly I can't take full ownership of this project. My goal is to make it works with .NET MAUI and don't block devs that depend on this package on XF. That said, if I manage to get this working and open a PR can you generate a nuget package?

**Comment by molinch** [comment]
Definitely! Thanks @pictos

**Comment by davidedwardsdev** [comment]
Hi. Is the Maui version working now? Can't seem to find it on Nuget

**Comment by wirejp** [comment]
Hi, any update on the nuget package for .Net Maui?

**Comment by Felicity-R** [comment]
Hi @pictos, now that Maui is in GA, do you still have plans to get the compatibility package working?

**Comment by pboisso** [comment]
Would love a Maui version too.

**Comment by giuseppenovielli** [comment]
Hello everyone!
Say thanks to @Redth for releasing a version for MAUI!
nuget -> https://www.nuget.org/packages/FFImageLoadingCompat.Maui
github -> https://github.com/Redth/FFImageLoading.Compat

**Comment by IainS1986** [comment]
Is that usable with just native .net-ios and .net-android or is it full MAUI only?

**Comment by fdhsdrdark** [comment]
@IainS1986 I would also be interested in the above question. FFImageLoading has some cool features other Image packages currently do not support.

**Comment by giuseppenovielli** [comment]
@IainS1986 @fdhsdrdark I don't test, but if i check into GituHub repository i think that there are all original code migrate to MAUI also only native net Android and ios

nuget -> https://www.nuget.org/packages/FFImageLoadingCompat
